src/main/java/org/openpnp/spi/Machine.java

```diff
@@ -163,6 +163,10 @@
 
     public boolean getHomeAfterEnabled();
 
+    public boolean getUsePickRotationInsteadOfRotationInTapeForStripFeeders();
```

Comment:
As I said before, this does not belong on Machine. The property should be on the strip feeder it belongs to. If you want a global default, you can use machine.setProperty() and machine.getProperty() from the ReferenceStripFeederWizard.

Reply:
ok, i didnt know about machine.setProperty() and machine.getProperty(), will fix it

Reply:
Can this be renamed to something that is not a sentence? like "isPickRotation()"